_data/projects/kwik.yml

```
@@ -0,0 +1,10 @@
+name: Kwik
+desc: A property-based testing library for Kotlin
+site: https://kwik.readthedocs.io
+tags:
+  - kotlin
+  - libraries
+  - testing
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/jcornaz/kwik/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22
```

There's a nicer link that can be used here:

```suggestion
  link: https://github.com/jcornaz/kwik/labels/up%20for%20grabs
```

---

Indeed. Thanks.